mm2src/coins/tendermint/tendermint_coin.rs

```diff
@@ -286,13 +287,10 @@ impl TendermintActivationPolicy {
                         .ok_or_else(|| io::Error::new(io::ErrorKind::InvalidData, "Couldn't generate public key"))
                 },
 
-                PrivKeyPolicy::Trezor => Err(io::Error::new(
+                _ => Err(io::Error::new(
                     io::ErrorKind::Unsupported,
-                    "Trezor is not supported yet!",
+                    "UnsupportedModeForPrivKeyPolicy",
```

Comment:
I would avoid the wildcard pattern to get compilation errors when adding new variants and remember handling them.

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2223/commits/3db29387107661f5402f14f0c48d441b20a62e26